templates/conf.d/nginx.conf.erb

```diff
@@ -330,5 +330,19 @@ stream {
 <% unless @confd_only -%>
   include <%= @conf_dir %>/streams-enabled/*;
 <% end -%>
+
+<% unless @stream_log_format.empty? -%>
```

Comment:
This usually renders poorly, but I'd do it the other way around: first the `if` or `unless`, then the empty line. That way you don't end up with a lot of empty lines if the block doesn't render.
```suggestion
<% unless @stream_log_format.empty? -%>

```